app/views/layouts/application.html.erb

```diff
@@ -125,50 +125,74 @@
       </div>
     </main>
 
-    <footer class="govuk-footer govuk-footer--app" role="contentinfo">
-      <div class="govuk-width-container ">
-
-        <div class="govuk-footer__meta">
-          <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
-            <h2 class="govuk-visually-hidden">Support links</h2>
-            <div class="govuk-footer__meta-custom">
-              <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-half">
-                  <h2 class="govuk-heading-s">
-                    Get support
-                  </h2>
-                  <ul class="govuk-footer__meta-custom govuk-list govuk-!-font-size-16">
-                    <li>Email: <%= support_email(subject: "Register trainee teachers support", classes: "govuk-footer__link") %></li>
-                    <li>We aim to respond within 5 working days, or one working day for more urgent queries</li>
-                  </ul>
-                </div>
-                <div class="govuk-grid-column-one-half">
-                  <h2 class="govuk-heading-s">
-                    Give feedback
-                  </h2>
-                  <p><%= govuk_link_to "Give feedback to help us improve Register trainee teachers", Settings.feedback_link_url, class: "govuk-footer__link" %></p>
-                </div>
-              </div>
-            </div>
-
-            <ul class="govuk-footer__inline-list govuk-!-margin-bottom-0">
-              <li class="govuk-footer__inline-list-item">
+    <footer class="govuk-footer">
```

Comment:
Any chance we can use https://govuk-components.netlify.app/components/footer/#footer-with-navigation instead please :pray:

Reply:
Good shout f49ecdc24ad7e90a286f8894bf17bf791f9af711